src/controllers/employer.ts

```diff
@@ -65,6 +65,44 @@ export const registerEmployer = async (
   }
 }
 
+//**Update employer*/
+export const updateEmployer = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    const employerId = req.params.id
```

Comment:
`employerId` currently has the type of `string`, but `findOne` method requires id to have type of `number`.